### DISCUSSION TITLE
New GUI-Beta requires Status DBus path for Temp sensors

### DISCUSSION THREAD

**Discussion by gharshaw** [freakent/dbus-mqtt-devices#80] [General]
@freakent  
Hi Martin, While reviewing the new gui-beta in v3.20~26 at  http://ipaddress/gui-beta I discovered that gui-beta is using a  "Status" dbus-path for temperature sensors. even though the documentation, https://github.com/victronenergy/venus/wiki/dbus#temperatures,  says this path is for "wired inputs only.   Without the status path  the "Environment" screen is blank.

I added the Status path to the services.yml like so --
  Status:
    description: "0=Ok, 1=Disconnected, 2=Short circuited, 3=Reverse polarity, 4=Unknown"
    persist: true
    default: 0
    min: 0
    max: 4

![image](https://github.com/freakent/dbus-mqtt-devices/assets/79295813/d1c69e98-20c5-4161-a400-248229f11586)

Best Regards,
Gene

**Comment by freakent** [comment]
Thanks for the info. ~~Where in the UI you posted do you see the status?~~ OK, I just re-read your message. Given that it has four possible values, is there one in particular that makes it display?

I don't think persist should be set to true.

**Comment by gharshaw** [comment]
@freakent , I defaulted it to 0. 0=Ok
I agree with your thoughts on persist. I'll set it to false and see how the app behaves.

I also added Status to the tank sensors

**Comment by freakent** [comment]
If you change "persist", make sure you use a different device or service name in the registration so that it creates a new set of settings on the dbus.

**Comment by johanvancalster** [comment]
I got the same issue with the new beta version. in the old gui and beta gui.
Nothing shows, exact samen code works perfect on release version of Venus OS.
MQQT explorer gives the result fine of level and remaining, but gui stays blank.
Anything else I can try.

**Comment by freakent** [comment]
I'm sorry this issue has been open for such a long time, and was origionally related to the Status attribute. Can you summarise what the problem is your are experiencing currently? If it's not related to the status attribute then pleae open a new issue.

**Comment by johanvancalster** [comment]
I'm testing out the beta release of Venus OS, to give the beta gui a try.
When using my working code to publish tank level, I get empty fields see below.
I added the status field like above, but still get empty values.

![Screenshot 2024-01-08 at 17 54 50](https://github.com/freakent/dbus-mqtt-devices/assets/125272616/0fa63988-657f-4e08-870e-f5f4fc67c200)
![Screenshot 2024-01-08 at 17 55 57](https://github.com/freakent/dbus-mqtt-devices/assets/125272616/71a71ee1-7315-425f-b638-a342ba82c82c)

**Reply by freakent** [reply]
@johanvancalster Can you retry with v0.6.4 I released this week for Venus v3.20?
